src/typing/typeloadFields.ml

```diff
@@ -1101,6 +1101,11 @@ let create_method (ctx,cctx,fctx) c f fd p =
 	let parent = (if not fctx.is_static then get_parent c (fst f.cff_name) else None) in
 	let dynamic = List.mem_assoc ADynamic f.cff_access || (match parent with Some { cf_kind = Method MethDynamic } -> true | _ -> false) in
 	if fctx.is_inline && dynamic then error (fst f.cff_name ^ ": You can't have both 'inline' and 'dynamic'") p;
+	let is_override = match fctx.override with
+		| None -> false
+		| _ -> true in
+	if (is_override && fctx.is_static) then error (fst f.cff_name ^ ": You can't have both 'override' and 'static'") p;
```

Comment:
This "You can't have both …" formulation seems a bit strange/informal. How about "`X` and `Y` cannot be applied to the same method" or something along those words?

Reply:
That on the other hand might be a bit _too_ formal. :) Maybe something like:

> `override` is not allowed on `static` functions

Reply:
Doing a quick Ctrl+Shift+F, "is not allowed on" seems to be a common pattern for Haxe error messages anyway.